src/cluster.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import * as MapboxGL from 'mapbox-gl';
+import { Props as MarkerProps } from './marker';
+
+const supercluster = require('supercluster'); //tslint:disable-line
+
+export interface Props {
+  ClusterMarkerFactory: (coordinates: number[], pointCount: number) => JSX.Element;
+  clusterThreshold?: number;
+  radius?: number;
+  maxZoom?: number;
+  minZoom?: number;
+  extent?: number;
+  nodeSize?: number;
+  log?: boolean;
+  children?: Array<React.Component<MarkerProps, void>>;
```

Comment:
I am using `array-simple` already but `React.Component<MarkerProps, void>` is not a simple type